components/sendinblue/actions/add-or-update-contact/add-or-update-contact.mjs

```diff
@@ -0,0 +1,101 @@
+import sendinBlueApp from "../../sendinblue.app.mjs";
+
+// legacy_hash_id: a_0Mi7n5
```

Comment:
This was a special comment we used when we converted a set of actions from our previous version to the latest component action API, but you won't need to include this in any new components.